Module4/3_Functions_several_arguments/2.recursivity.py

```diff
@@ -0,0 +1,25 @@
+"""Functions recursivity"""
```

Comment:
This topic at first is a little bit dificult to understand but,  when you get it, it'll be easier and funny 😆

Reply:
Thanks 🐍